Packs/CTIX/Integrations/CTIXv3/README.md

```diff
@@ -1380,4 +1380,173 @@ Lookup to get threat data
               "valid_from": 1652081902,
               "valid_until": null}],
  "total": 1}
-```
\ No newline at end of file
+```
+
+
+
+## ctix-get-all-notes
```

Comment:
please generate the documentation for the new commands using `demisto-sdk generate-docs` command (use the -e flag for using a commad_examples file) - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.

when its done, you can append the docs for the new commands to this file.

Reply:
Should this be taking a long time to complete? When I try running this command, it just hangs endlessly before killing it, even with verbose mode enabled

Reply:
Fixed.